Resources/views/PlatformUI/shell.html.twig

```diff
@@ -52,33 +52,34 @@
             var app = new Y.eZ.PlatformUIApp({
                     container: '.ez-platformui-app',
                     viewContainer: '.ez-view-container',
-                    root: "{{ app.request.attributes.get( 'semanticPathinfo' ) }}",
-                    assetRoot: "{{ asset( '/' ) }}",
-                    anonymousUserId: "{{ anonymousUserId }}",
+                    config: {{ parameters|json_encode|raw }},
```

Comment:
Oh, you meant the addition of the variable. Okay, I'll remove it. I've looked into the JS code, and I really can't change that without a bit of training first :-)